v3-docs/docs/.vitepress/config.mts

```diff
@@ -149,6 +149,10 @@ export default defineConfig({
             text: "Roadmap",
             link: "/about/roadmap",
           },
+          {
+            text: "Contributing",
+            link: "/about/what-is#contributing",
```

Comment:
It’s a bit confusing in the [docs preview](https://deploy-preview-13991.docs-online.meteor.com/about/what-is.html) since Contributing also appears under `/about/what-is`, and both `/about/what-is` and `/about/what-is#contributing` show up in the nav menu. Shouldn’t we move everything to its own page instead, like `/CONTRIBUTING.md` or `/about/contributing.md`?

Maybe we can include in its own section both the contribution info in the OSS tool project itself and in the documentation.

Reply:
I suggest adding a dedicated page for `Contributing` after `Roadmap` in the menu, and move content from `v3-docs/docs/CONTRIBUTING.md` inside.

Also move meteor ressources to nested `items` to fix bottom page navigation. 
<img width="235" height="170" alt="Capture d’écran 2025-10-20 à 15 08 53" src="https://github.com/user-attachments/assets/3ca66223-e156-4398-a883-6d9e4967f9dd" />

Does that sounds good to you?